Makefile

```diff
@@ -67,4 +67,14 @@ distclean:
 	rm -rf tmp_install
 	cargo clean
 
+fmt:
```

Comment:
Could you mark this target as [phony](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)? E.g.

```make
.PHONY: fmt
fmt:
    ...
```

Reply:
done